crates/cli/src/subcommands/generate/csharp.rs

```diff
@@ -712,6 +712,8 @@ pub fn autogen_csharp_globals(items: &[GenItem], namespace: &str) -> Vec<(String
                     );
                 }
                 writeln!(output, "\"<none>\" => null,");
+                writeln!(output, "\"__identity_disconnected__\" => null,");
```

Comment:
Added it

Reply:
FWIW in modules we treat all events that start with `__` and end with `__` as "special" in various places.

Could do the same here instead of hardcoding a few names.